static/build_html.sh

```diff
@@ -5,16 +5,16 @@ set -e
 # Read arguments
 BRANCH_NAME=$1
 JOB_NAME=$2
+REPO_NAME=$3
 
 STABLE_BASE_PATH="https://d2l.ai"
 PREVIEW_BASE_PATH="http://preview.d2l.ai/$JOB_NAME"
-PREVIEW_BASE_PATH_RELEASE="${PREVIEW_BASE_PATH/release/master}"
+alternate_base="http://preview.d2l.ai/$REPO_NAME/master"
 
 # Generate Headers
 if [[ "$BRANCH_NAME" == "release" ]]; then
     echo "Use release headers"
     alternate_text="Preview Version"
```

Comment:
```suggestion
    alternate_text="Preview Version"
    alternate_base="${PREVIEW_BASE_PATH/release/master}"
```

Reply:
This is what I meant. Otherwise your line 12 `alternate_base="http://preview.d2l.ai/$REPO_NAME/master"` will be overwritten by line 22 `alternate_base=$STABLE_BASE_PATH` in that branch: this may be confusing

Reply:
Got it, thanks for the changes.

Reply:
It is still overwritten. No?

Reply:
Ok sorry, i don't know what happened. Seems like batched applied suggestions did not work correctly.